core/ledger/snapshotgrpc/snapshot_service.go

```diff
@@ -84,6 +103,33 @@ func (s *SnapshotService) QueryPendings(ctx context.Context, signedRequest *pb.S
 	return &pb.QueryPendingSnapshotsResponse{BlockNumbers: result}, nil
 }
 
+func (s *SnapshotService) checkACL(channelID string, resName string, signatureHdr *cb.SignatureHeader, signedRequest *pb.SignedSnapshotRequest) error {
+	if signatureHdr == nil {
+		return errors.New("missing signature header")
+	}
+
+	expirationTime := crypto.ExpiresAt(signatureHdr.Creator)
```

Comment:
Just to double check: this ACL evaluation is not perfomed on the commit path right? Otherwise a time check like this would get us to fork.

Reply:
Same as above.